include/tue/config/map.h

```diff
@@ -36,6 +36,7 @@ class Map : public Node
         return true;
     }
 
+    // addGroup never returns false, therefore overwriting is possible. Use readGroup before to prevent overwriting
```

Comment:
So why do we still need this (and/or is it exposed)?

Reply:
This is on the node level. This is for internal use. The question is: Do you want to be able to overwrite stuff. Because it isn't possible to remove data at the moment. But on the writer level, `readGroup` is called before `addGroup`. `addGroup` is never called, when `readGroup` returns true.